### PR TITLE
2815: typos on CRUD page

### PR DESCRIPTION
**Pull request by macMikey**
see https://github.com/serverpod/serverpod/discussions/2815

### CODE REVIEW

File: docs/06-concepts/06-database/05-crud.md [hunk 1]
```diff
@@ -128,15 +128,15 @@ Update a single row, the following code will update the company name, but will n
 var company = await Company.db.findById(session, companyId); 
 company.name = 'New name';
 company.address = 'Baker street'
```
Comment (hampuslavin) [inline docs/06-concepts/06-database/05-crud.md:130]: Would you mind adding the missing semicolon here as well? 🙏

**Review by hampuslavin — COMMENTED** [review]
Thanks! 🎉 🚀 One small extra thing I saw now if you don't mind.

File: docs/06-concepts/06-database/05-crud.md (hunk 1 above)
Comment (macMikey) [reply docs/06-concepts/06-database/05-crud.md:130]: of course! i'm still a n00b at dart/flutter/serverpod, so i appreciate the help, and the opportunity to help.

**Review by Isakdl — APPROVED** [review]
LGTM 🚀 

Thank you :)

**Pull request merged**